### ISSUE TITLE
Build Error when building for Universal

### ISSUE THREAD

**Issue by Joniras** [auth0/angular2-jwt#636]
### Description

When building the server (ng run capp:server) with the configuration (from angular.json): 

```json
{
   "server":{
      "builder":"@angular-devkit/build-angular:server",
      "options":{
         "stylePreprocessorOptions":{
            "includePaths":[
               "./src/sass"
            ]
         },
         "outputPath":"dist/server",
         "main":"src/main.server.ts",
         "tsConfig":"src/tsconfig.server.json",
         "fileReplacements":[
         ]
      }
   }
}
``` 

tsconfig.server.json:
```json
{
   "extends":"../tsconfig.json",
   "compilerOptions":{
      "outDir":"../out-tsc/app",
      "baseUrl":".",
      "module":"commonjs",
      "types":[
         "node"
      ],
      "paths":{
         "@APIServer/*":[
            "../../server/src/*"
         ],
         "@shared/*":[
            "app/shared/*"
         ]
      }
   },
   "angularCompilerOptions":{
      "entryModule":"app/app.server.module#AppServerModule"
   },
   "files":[
      "main.server.ts"
   ]
}
```

It produces the error: 
![image](https://user-images.githubusercontent.com/8112178/74154108-4ccf5580-4c12-11ea-8553-540041f60554.png)
`ERROR in node_modules/@auth0/angular-jwt/lib/angular-jwt.module.d.ts:15:22 - error NG6002: Appears in the NgModule.imports of AppModule, but could not be resolved to an NgModule class`
    
Building works when removing JWTModule from Imports.
### Reproduction
The error is consistent (appears every time i try to build and can be reproduced when building a client for SSR) with the given configuration.

Importing can happen with any parameter (configuration).

I already tried deleting node_modules/ and installing everything new, running ivy-ngcc (ngcc) manually as well as different JWTModule.forRoot() configurations.

### Environment

- "@auth0/angular-jwt": "^4.0.0"
- "@angular-devkit/build-angular": "~0.900.1"
- "@angular/common": "^9.0.0"
- newest Angular Version (9.0.0)

### Suggestions

I read in other Issues on Github with a similar Problem that it may be due to metadata-file placement in the library.
Another mentions the not yet given compatibility with ivy for libraries and to use the old View-Engine for packaging.

**Comment by Joniras** [comment]
Ok if i downlaod the repo, build the package myself and place it in the node_modules of the app, everything works fine.

Maybe it just needs to be published again with the newest versions ?

**Comment by Joniras** [comment]
The only differences in the build that were visible to me were:

> auth0-angular-jwt.d.ts
```ts
...
//# sourceMappingURL=auth0-angular-jwt.d.ts.map // new
```

> angular-jwt.module.d.ts
```ts
import * as ɵngcc0 from '@angular/core'; // new
... 
export declare class JwtModule {
    constructor(parentModule: JwtModule);
    static forRoot(options: JwtModuleOptions): ModuleWithProviders<JwtModule>;
    static ɵmod: ɵngcc0.ɵɵNgModuleDefWithMeta<JwtModule, never, never, never>; // new
    static ɵinj: ɵngcc0.ɵɵInjectorDef<JwtModule>; //new
}

//# sourceMappingURL=angular-jwt.module.d.ts.map // new
```
## TL;DR
SourceMaps and NGCC additions

**Comment by Joniras** [comment]
The only differences in the build that were visible to me were:


> angular-jwt.module.d.ts
```ts
import * as ɵngcc0 from '@angular/core'; // new
... 
export declare class JwtModule {
    constructor(parentModule: JwtModule);
    static forRoot(options: JwtModuleOptions): ModuleWithProviders<JwtModule>;
    static ɵmod: ɵngcc0.ɵɵNgModuleDefWithMeta<JwtModule, never, never, never>; // new
    static ɵinj: ɵngcc0.ɵɵInjectorDef<JwtModule>; //new
}

//# sourceMappingURL=angular-jwt.module.d.ts.map // new
```


> jwthelper.service.d.ts
```ts
import * as ɵngcc0 from '@angular/core'; //new
...
export declare class JwtHelperService {
    ...
    static ɵfac: ɵngcc0.ɵɵFactoryDef<JwtHelperService>;
    static ɵprov: ɵngcc0.ɵɵInjectableDef<JwtHelperService>;
}
```

> jwt.interceptor.d.ts
```ts
...
import * as ɵngcc0 from '@angular/core'; //new
export declare class JwtInterceptor implements HttpInterceptor {
    ...
    static ɵfac: ɵngcc0.ɵɵFactoryDef<JwtInterceptor>; // new
    static ɵprov: ɵngcc0.ɵɵInjectableDef<JwtInterceptor>; // new
}
...
//# sourceMappingURL=jwt.interceptor.d.ts.map // new
```

## TL;DR
NGCC additions

**Comment by Joniras** [comment]
Ok i did this to myself: 
 - as proposed (by me) in #30, i had a postinstall script to babel-compile @autho0/angular-jwt
 - this appearently broke the module

## Solution
 remove the postinstall script to babel-compile the package

to whoever is reading this ever and had the same problem and i was the cause: I'm sorry 😬

**Comment by stevehobbsdev** [comment]
Thanks @Joniras ! Glad you found the issue :)
